sites.json

```diff
@@ -1666,11 +1666,14 @@
     },
 
     {
-        "name": "Discord",
+        "name": "Discord(app)",
         "url": "https://support.discordapp.com/hc/en-us/articles/212500837-How-do-I-permanently-delete-my-account-",
         "difficulty": "easy",
+        "notes":"You will need to transfer the ownership of your servers.",
```

Comment:
How about `If you're a server owner, you'll need to either <a href="https://support.discordapp.com/hc/en-us/articles/213595197-How-do-I-delete-a-server-">delete the server</a>, or <a href="https://support.discordapp.com/hc/en-us/articles/216273938-How-do-I-transfer-server-ownership-">transfer ownership</a> for account deletion to succeed.` instead? (adapted from the URL)

Reply:
Also, can a space be added after the colon, before the quote for the second string? eg. `"notes": " ... ",`

Reply:
That sounds good to me; I updated the notes.